### PR TITLE
Preface Update

### PR DESCRIPTION
**Pull request by mhoeber**
For XBlock doc changes, as well as style guide related changes

### CODE REVIEW

**Comment by mhoeber** [comment]
@lamagnifica

File: en_us/shared/preface.rst [hunk 1]
```diff
@@ -6,21 +6,17 @@ Preface
...
 
-Course teams, researchers, developers, students: the edX community includes
+Course teams, researchers, developers, learners: the edX community includes
 groups with a range of reasons for using the platform and objectives to
 accomplish. To help members of each group learn about what edX offers, reach
 goals, and solve problems, edX provides a variety of information resources.
 
-To help you find what you need, browse the edX offerings in these categories:
+To help you find what you need, browse the edX offerings in the following
+categories
```
Comment (lamagnifica) [inline en_us/shared/preface.rst]: add period.

File: en_us/shared/preface.rst [hunk 2]
```diff
@@ -234,24 +238,30 @@ Documentation for developers is available on the docs.edx.org web page.
   for getting an edX developer stack (Devstack) and production stack
   (Fullstack) operational.
 
-* XBlock_: Open edX courseware components provides preliminary documentation
-  on the XBlock component architecture for building courses.
+* `Open edX XBlock Tutorial`_: guides developers through the process of
+  creating an XBlock, and explains the concepts and anatomy of XBlocks.
+
+* `Open EdX XBlock API Guide`_: provides reference information on the XBlock
```
Comment (lamagnifica) [inline en_us/shared/preface.rst]: lc edX?

on --> about

File: en_us/shared/preface.rst [hunk 3]
```diff
@@ -234,24 +238,30 @@ Documentation for developers is available on the docs.edx.org web page.
   for getting an edX developer stack (Devstack) and production stack
   (Fullstack) operational.
 
-* XBlock_: Open edX courseware components provides preliminary documentation
-  on the XBlock component architecture for building courses.
+* `Open edX XBlock Tutorial`_: guides developers through the process of
```
Comment (lamagnifica) [inline en_us/shared/preface.rst]: the remaining list items do not include the colon as a separator

File: en_us/shared/preface.rst [hunk 4]
```diff
@@ -234,24 +238,30 @@ Documentation for developers is available on the docs.edx.org web page.
...
-* `edX Data Analytics API`_ provides tools for building applications to view
-  and analyze student activity in your course.
+* `edX Data Analytics API`_ provides reference information for using the data
+  analytics API to build applications to view and analyze learner activity in
+  your course.
 
-* `edX Platform APIs`_ provide tools for building applications to view course
-  information and videos and work with mobile apps, users, and enrollment.
+* `edX Platform APIs`_ provide reference information for building applications
+  to view course information and videos and worth with user and enrollment
```
Comment (lamagnifica) [inline en_us/shared/preface.rst]: ..and videos, and worth --> work?...

File: en_us/shared/preface.rst [hunk 5]
```diff
@@ -332,6 +343,28 @@ the most recently released additions to the platform.
...
+* `edX Open Learning XML Guide`_ provides guidelines for building edX courses
+  with Open Learning XML (OLX). Note that this guide is currently an alpha
+  version.
+
+* `edX Data Analytics API`_ provides reference information for using the data
+  analytics API to build applications to view and analyze learner activity in
+  your course.
+
+* `edX Platform APIs`_ provide reference information for building applications
+  to view course information and videos and worth with user and enrollment
```
Comment (lamagnifica) [inline en_us/shared/preface.rst]: worth again

File: en_us/shared/preface.rst [hunk 6]
```diff
@@ -332,6 +343,28 @@ the most recently released additions to the platform.
...
   features released after the most recent official, named release.
 
+* The `edX Platform Developer's Guide`_ includes guidelines for
+  contributing to Open edX, options for extending the Open edX platform, using
+  the edX public sandboxes, instrumenting analytics, and testing.
+
+* `Open edX XBlock Tutorial`_: guides developers through the process of
+  creating an XBlock, and explains the concepts and anatomy of XBlocks.
+
+* `Open EdX XBlock API Guide`_: provides reference information on the XBlock
```
Comment (lamagnifica) [inline en_us/shared/preface.rst]: lc edX, no colon

File: en_us/shared/preface.rst [hunk 7]
```diff
@@ -332,6 +343,28 @@ the most recently released additions to the platform.
 * `Building and Running an Open edX Course - latest`_ includes descriptions of
   features released after the most recent official, named release.
 
+* The `edX Platform Developer's Guide`_ includes guidelines for
+  contributing to Open edX, options for extending the Open edX platform, using
+  the edX public sandboxes, instrumenting analytics, and testing.
+
+* `Open edX XBlock Tutorial`_: guides developers through the process of
```
Comment (lamagnifica) [inline en_us/shared/preface.rst]: no colon

File: en_us/shared/preface.rst [hunk 8]
```diff
@@ -332,6 +343,28 @@ the most recently released additions to the platform.
...
+  creating an XBlock, and explains the concepts and anatomy of XBlocks.
+
+* `Open EdX XBlock API Guide`_: provides reference information on the XBlock
+  API.
+
```
Comment (lamagnifica) [inline en_us/shared/preface.rst:355]: the edX in the remaining three bullets should by uc: EdX

File: en_us/shared/preface.rst [hunk 9]
```diff
@@ -332,6 +343,28 @@ the most recently released additions to the platform.
...
+* `edX Open Learning XML Guide`_ provides guidelines for building edX courses
+  with Open Learning XML (OLX). Note that this guide is currently an alpha
```
Comment (lamagnifica) [inline en_us/shared/preface.rst:357]: is the guide an alpha version, or does the guide describe an alpha version of OLX?

**Comment by lamagnifica** [comment]
please remember to update the preface in the edx-analytics-dashboard repo as well.
:+1: after spelling error change

**Pull request merged**